docs/operate/control/headless-app.md

```diff
@@ -5,5 +5,532 @@ weight: 30
+
+        # wait before checking again
+        await asyncio.sleep(10)
+
+if __name__ == '__main__':
+    asyncio.run(main())
+```
+
+{{< /expand >}}
+
+## Run your script
+
+You can run your script manually from your terminal, or you can automatically run it each time your machine starts.
+
+### Run your script manually
+
+To run your script on a laptop or desktop, or to run it on your machine's SBC manually for testing purposes, execute the following command in a terminal:
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+python3 /home/myName/project/my_cool_script.py
+```
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+go run /home/myName/project/my_cool_script.go
+```
+
+{{% /tab %}}
+{{% tab name="C++" %}}
+
+For information on running C++ code see [the instructions on GitHub](https://github.com/viamrobotics/viam-cpp-sdk/blob/main/BUILDING.md).
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Run your script as an automatic process
+
+If you want your script to run each time your machine boots, configure the script as a _{{< glossary_tooltip term_id="process" text="process" >}}_ on your machine.
+Configured processes are managed by `viam-server` and are a way to run any specified command either once on boot or continuously over the lifecycle of `viam-server`.
+
+{{% alert title="Tip" color="tip" %}}
+If you are running your code from a laptop or desktop, we do not recommend configuring your script to run as a process because doing so will cause the process to run whenever you boot your computer, even when you are using your computer for unrelated purposes.
+{{% /alert %}}
+
+To configure a process, click the **+** button on your machine's **CONFIGURE** tab and select **Process**.
+For detailed configuration information, see [Configure a managed process](/manage/reference/processes/).
+
+#### Example
+
+The following example executes the command `python3 my_cool_script.py` in your <file>/home/myName/project/</file> directory every time your machine boots, and restarts the process if it stops running.
+
+{{< tabs >}}
+{{% tab name="Builder mode" %}}
+
+![The CONFIGURE tab with a process called run-my-code configured. The executable is python3, the argument is my_cool_script.py, and the working directory is /home/myName/project. Logging is turned on and execute once is turned off.](/build/configure/process-fancy.png)
+
+{{% /tab %}}
+{{% tab name="JSON" %}}
+
+```json
+"processes": [
+    {
+      "id": "run-my-code",
+      "log": true,
+      "name": "python3",
+      "args": [
+        "my_cool_script.py"
+      ],
+      "cwd": "/home/myName/project/"
+    }
+  ]
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## Debug
+
+Read and filter a machine's logs to view updates from your machine's `viam-server` instance and troubleshoot issues with your program.
+
+{{< tabs >}}
+{{% tab name="App UI" %}}
```

Comment:
UI here is outdated. you can pull the newer video of log filtering here, take a new screenshot or remove the image

Reply:
Took a new screenshot